### DISCUSSION TITLE
tutorials/networking/webrtc

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#189] [User-contributed notes]
# tutorials/networking/webrtc

HTML5, WebSocket, WebRTC: One of Godot's great features is its ability to export to the HTML5/WebAssembly platform, allowing your game to run directly in the browser when a user visit your webpage....

https://docs.godotengine.org/en/stable/tutorials/networking/webrtc.html

**Comment by jyotiroy-jrs** [comment]
Need webrtc android library for arm32 architecture, debug and release.

**Reply by Calinou** [reply]
Please open an issue on the webrtc-native repository, as feature requests should go there: https://github.com/godotengine/webrtc-native
